### PR TITLE
esphome: remove dotlambda from maintainers

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/aioesphomeapi/default.nix [hunk 1]
```diff
@@ -22,6 +22,6 @@ buildPythonPackage rec {
     description = "Python Client for ESPHome native API";
     homepage = "https://github.com/esphome/aioesphomeapi";
     license = licenses.mit;
-    maintainers = with maintainers; [ dotlambda ];
+    maintainers = with maintainers; [ fab ];
```
Comment (fabaff) [inline pkgs/development/python-modules/aioesphomeapi/default.nix:25]: Added myself as this is a pretty popular integration in Home Assistant.

**Review by fabaff — APPROVED** [review]

File: pkgs/tools/misc/esphome/default.nix [hunk 2]
```diff
@@ -49,6 +49,6 @@ in python.pkgs.buildPythonApplication rec {
     description = "Make creating custom firmwares for ESP32/ESP8266 super easy";
     homepage = "https://esphome.io/";
     license = licenses.mit;
-    maintainers = with maintainers; [ dotlambda globin elseym ];
+    maintainers = with maintainers; [ globin elseym ];
```
Comment (Mic92) [inline pkgs/tools/misc/esphome/default.nix:52]: How about you? @makefu

Comment (mweinelt) [reply pkgs/tools/misc/esphome/default.nix:52]: Are you still interested in maintaining esphome @globin?

**Comment by mweinelt** [comment]
I have quite some esphome devices in play, so here we are.

**Pull request merged**